source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/__tests__/acceptance-test.tsx

```diff
@@ -61,66 +20,95 @@ describe.only('Acceptance | Archive Project', () => {
+            organization: { data: { id: 1, type: 'organizations' } }
+          }
+        },
+        included: [
+          { type: 'organizations', id: 1, }
+        ]
       });
+    });
 
-      describe('archive project', () => {
+    describe('archiving the project', () => {
+      beforeEach(function () {
+        this.mockPatch(200, 'projects/1', {
+          data: {
+            type: 'projects',
+            id: '1',
+            attributes: {
+              'date-archived': "2018-08-10T23:59:55.259Z"
+            },
+        }});
+      });
 
-        beforeEach(async function () {
-          this.mockPatch(200, 'projects/1', {
-            data: {
-              type: 'projects',
-              id: '1',
-              attributes: {
-                'name': "Dummy Project",
-                'type': "Sogdian Transalation",
-                'description': "This is a dummy project",
-                'language': "Sogdian",
-                'private': false,
-                'date-created': "2018-08-09T20:23:54.809962",
-                'date-updated': "2018-08-10T23:59:55.7694263Z",
-                'date-archived': "2018-08-10T23:59:55.259Z"
-              },
-              relationships: {}
-          });
+      describe('the archive button is clicked', () => {
+        beforeEach(async () => {
+          await visit('/project/1');
+          await page.clickArchiveLink();
         });
 
-        it("it's archived", () => {
+        it("changes the button text", () => {
           expect(page.archiveText).to.equal('Reactivate');
         });
+      });
+    });
+  });
 
-        describe('reactivate project', () => {
+  describe('an archived project exists', () => {
+    beforeEach(function() {
+      this.mockGet(200, 'projects/1', {
+        data: {
+          type: 'projects',
+          id: '1',
+          attributes: {
+            'date-archived': "2018-08-10T23:59:55.259Z"
+          },
+          relationships: {
+            organization: { data: { id: 1, type: 'organizations' } }
+          }
+        },
+        included: [
+          { type: 'organizations', id: 1, }
+        ]
+      });
+    });
 
-          beforeEach(async function () {
-            this.mockPatch(200, 'projects/1', {
-              data: {
-                type: 'projects',
-                id: '1',
-                attributes: {
-                  'name': "Dummy Project",
-                  'type': "Sogdian Transalation",
-                  'description': "This is a dummy project",
-                  'language': "Sogdian",
-                  'private': false,
-                  'date-created': "2018-08-09T20:23:54.809962",
-                  'date-updated': "2018-08-10T23:59:55.7694263Z",
-                  'date-archived': null
-                },
-                relationships: {}
-              });
-          });
+    describe('reactivating a project', () => {
+      beforeEach(function () {
+        this.mockPatch(200, 'projects/1', {
```

Comment:
how is that the page loads if it doesn't have the other parameters?

Reply:
They end up just not being rendered. Because react doesn't render undefined values